samples/java/src/test/java/com/smartbear/readyapi4j/samples/java/ApiTestBase.java

```diff
@@ -0,0 +1,50 @@
+package com.smartbear.readyapi4j.samples.java;
+
+import com.smartbear.readyapi.client.TestRecipe;
+import com.smartbear.readyapi.client.execution.Execution;
+import com.smartbear.readyapi.client.execution.ProjectExecutionRequest;
+import com.smartbear.readyapi.client.execution.ProjectExecutor;
+import com.smartbear.readyapi.client.execution.RecipeExecutor;
+import com.smartbear.readyapi.client.execution.TestServerClient;
+import com.smartbear.readyapi.client.support.AssertionUtils;
+import org.junit.BeforeClass;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.MalformedURLException;
+
+public class ApiTestBase {
+
+    final static Logger LOG = LoggerFactory.getLogger(ApiTestBase.class);
+    private static TestServerClient testServerClient;
+    private static RecipeExecutor recipeExecutor;
+    private static ProjectExecutor projectExecutor;
+
+    @BeforeClass
+    public static void initExecutor() throws MalformedURLException {
+        String hostName = System.getProperty("testserver.host", "http://testserver.readyapi.io:8080");
```

Comment:
this while initialisation shouldn't be needed - it is done in the RecipeExecutorFacade